repose-aggregator/services/phone-home/src/main/scala/org/openrepose/core/services/phonehome/PhoneHomeService.scala

```diff
@@ -127,7 +128,23 @@ class PhoneHomeService @Inject()(@Value(ReposeSpringProperties.CORE.REPOSE_VERSI
         }
       }
 
+      def getCurrentFormattedDateTime(): String = {
+        // From http://docs.oracle.com/javase/7/docs/api/java/text/SimpleDateFormat.html
+        // "For formatting, if the offset value from GMT is 0, 'Z' is produced."
+        // This only manipulates the Time Zones that produce a 'Z'.
+        val formattedString = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSXXX").format(new Date())
+        if (formattedString.endsWith("Z")) {
+          formattedString.substring(0, formattedString.length() - 1) + "+00:00"
+        } else {
+          formattedString
+        }
+      }
+
       val updateMessage = Json.stringify(Json.obj(
+        "createdAt" -> getCurrentFormattedDateTime,
+        "createdAtMillis" -> System.currentTimeMillis,
```

Comment:
The createdAt and createdAtMillis values could differ here. You may want to record the time once by calling System.currentTimeMillis, then pass that value to the new Date(...). This would be an exceptionally rare case, but we could be 1ms off across formats.